core/src/main/java/org/elasticsearch/search/sort/ScriptSortBuilder.java

```diff
@@ -189,7 +112,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params builderParams)
             builder.field("nested_path", nestedPath);
         }
         if (nestedFilter != null) {
-            builder.field("nested_filter", nestedFilter, params);
```

Comment:
Were the `params` here a mistake before? If not, with non-null params this would have rendered to an array, which I think the parse would not accept. Looking at older commits (e.g. 10ec2e948a2f1426a5058ca3b2c2e39952f141d8), this looks like once where the `builderParams` but this was mixed up with the script params at some point.